doc/http-api/backend_swagger.yml

```diff
@@ -220,7 +220,7 @@ paths:
           required: false
           type: integer
       description: |
-        Retrieve recent message from my archive.
+        Retrieve the recent message from my archive.
```

Comment:
We shouldn't have any `my`s in this file because it's not documentation for client developers (there's no _my_ in issuing administration calls).